sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubAsyncClient.java

```diff
@@ -95,6 +98,15 @@
             .setScheduler(connectionOptions.getScheduler());
     }
 
+    /**
+     * Gets the Event Hub name this client interacts with.
+     *
+     * @return The Event Hub name this client interacts with.
+     */
+    public String getEventHubName() {
```

Comment:
Do we want to make this public?

Reply:
Yes. In one of our stand-ups every other language mentioned they had this exposed, so we were the only ones who did not.

Reply:
For example, .NET has this public:

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs#L30-L42